salt/core/firewall/files/firewall_reset.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# This file is managed by Dogsbody Technology Ltd.
+#   https://www.dogsbody.com/
+#
+# Description:  A script to reset iptables rules.
+#
+# Usage:  $0
+#
+
+iptables -P INPUT ACCEPT
```

Comment:
Should this file `set -euo pipefail`?

Also, should it `source /home/sysadmin-tools/firewall-settings.local` to use variables for the command paths like in `firewall.sh`, or is that generally unnecessary?

Reply:
I think the most likely scenario is someone makes a commit that introduces an error. Without `set -euo pipefail`, the command that errors will be silent, and it'll appear like the script succeeded. I think it's better to have feedback.

Since this script is always manually run while logged into the server, there's not much consequence to terminating early, as far as I can see.

Reply:
Added